### DISCUSSION TITLE
Clase de semana santa y QMP

### DISCUSSION THREAD

**Discussion by AlejoGol** [dds-jv/foro#339] [Consultas generales] [answered]
Hola, quería preguntar, va a haber clases el jueves que viene siendo semana santa?
Y si es que no, la primer iteración de QMP seria para este jueves o para el siguiente en el que tendríamos clase?
Saludos

**Comment by gastonprieto** [comment, marked as answer]
Buenas @AlejoGol 

Este jueves santo no hay clases. La entrega sería para la oróxima clase (semana siguiente).
